### PR TITLE
Add server type to resource card, fix windows/app launch buttons

### PR DESCRIPTION
**Pull request by avatus**
A couple small fixes.
First, adding the addr/tunnel distinction back to servers in the Unified Resource cards. 
Example:
Screenshot 2023-08-28 at 2 55 42 PM
If not a tunnel, we show the addr like before. Also, if its an openssh server, the text will be `OpenSSH Server`

Fixes a bug for longer login launch names in the MenuLogin and the uppercase LAUNCH
Bug:
image (2)

Fix:
Screenshot 2023-08-28 at 3 24 05 PM

### CODE REVIEW

File: lib/web/ui/server.go [hunk 1]
```diff
@@ -42,6 +42,8 @@ type Server struct {
 	Kind string `json:"kind"`
 	// Tunnel indicates of this server is connected over a reverse tunnel.
 	Tunnel bool `json:"tunnel"`
+	// OpenSSh returns true for OpenSSHNode and OpenSSHEICENodes
```
Comment (zmb3) [inline lib/web/ui/server.go:45]:
```suggestion
	// OpenSSH returns true for OpenSSHNode and OpenSSHEICENodes
```

But should we just send subkind instead? With this approach, we'll have to add a new bool every time we add a new subkind.

**Review by zmb3 — APPROVED** [review]

File: lib/web/ui/server.go (hunk 1 above)
Comment (avatus) [reply lib/web/ui/server.go:45]: Sure, sounds better. I wasn't sure how many subkinds we had but in this case it'll just set us up for the future anyway.

**Review by flyinghermit — APPROVED** [review]

**Review by zmb3 — APPROVED** [review]

File: lib/web/ui/server.go [hunk 2]
```diff
@@ -93,7 +95,13 @@ func MakeServer(clusterName string, server types.Server, accessChecker services.
 		Addr:        server.GetAddr(),
 		Tunnel:      server.GetUseTunnel(),
 		SSHLogins:   serverLogins,
-	}, nil
+	}
+
+	if server.IsOpenSSHNode() {
+		uiServer.SubKind = "OpenSSH Server"
```
Comment (zmb3) [inline lib/web/ui/server.go:101]: Is there a server.GetSubKind() you can use instead?

Comment (avatus) [reply lib/web/ui/server.go:101]: 🤦 yes, updating

**Pull request merged**